src/transaction/log_manager.c

```diff
@@ -9757,6 +9760,20 @@ log_checkpoint_execute (cubthread::entry & thread_ref)
 }
 #endif /* SERVER_MODE */
 
+#if defined(SERVER_MODE)
+static void
+log_checkpoint_trantable_execute(cubthread::entry &thread_ref)
```

Comment:
Is there a problem with indent? It should automatically do this:
```suggestion
log_checkpoint_trantable_execute (cubthread::entry &thread_ref)
```